Makefile

```diff
@@ -28,6 +28,13 @@ update:
 	go get -t -v -d -u ./...
 	@echo "Dependencies updated"
 
+.PHONY: update
+update-schema:
+	go build -v -tags 'libsqlite3 devel' ./lxd/schema.go
```

Comment:
Ok, I see what the devel tag is below, though "go run" and "go build" will ignore such tags when passed a file directly.

So you could just run

    go run -v ${TAGS} ./lxd/schema.go

And change the build tag for "lxd/schema.go" to something like "// +build never" so that we don't accidentally attempt to build it and get a confusing go error because we suddenly have two main functions :)